Library/Models/InterviewerChanged.cs

```diff
@@ -0,0 +1,38 @@
+﻿//    This file is part of Nfield.SDK.
+//
+//    Nfield.SDK is free software: you can redistribute it and/or modify
+//    it under the terms of the GNU Lesser General Public License as published by
+//    the Free Software Foundation, either version 3 of the License, or
+//    (at your option) any later version.
+//
+//    Nfield.SDK is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//    GNU Lesser General Public License for more details.
+//
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with Nfield.SDK.  If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Nfield.Models
+{
+    /// <summary>
+    /// Holds key properties of the created interviewer
+    /// </summary>
+    public class InterviewerChanged
```

Comment:
The model **InterviewerChanged** is used to create, update and password update, so this name is not the best one.
I suggest:
- InterviewerResponse
- InterviewerIdentifier
- InterviewerKeys

Also you should change the summary to add ` create update or password update`